spikeextractors/cacherecordingextractor.py

```diff
@@ -0,0 +1,22 @@
+from spikeextractors.extractors.bindatrecordingextractor import BinDatRecordingExtractor
+import tempfile
+import os
+
+
+class CacheRecordingExtractor(BinDatRecordingExtractor):
+
+    extractor_name = 'CacheRecordingExtractor'
+
+    def __init__(self, recording, dtype=None, chunksize=None):
+        self._tmp_file = tempfile.NamedTemporaryFile(suffix=".dat").name
+        if dtype is None:
+            dtype = recording.get_traces(start_frame=0, end_frame=2).dtype
```

Comment:
why **end_frame=2** ? **end_frame=1** should be enought ? for security 2 samples are better :)